### DISCUSSION TITLE
How to config Field Values List for string-array-map type field

### DISCUSSION THREAD

**Discussion by billjliu** [sngular/kloadgen#337] [Q&A]
I am using the "KLG - Value Schema Serializer Config" to try to produce an event with some pre-defined data in a field of string-array-map type, but I could not figure out the syntax of the "Field Values List" for the string-array-map type. For example, I tried 
[key1:[value1, value2, value3], key2:[valueB1,valueB2]], but the kloadgen created the event with randomly created key names and took "key1:[value1", "value2" and "value3]", etc, as the values of in the string array. So the key name "key1" in my configuration was not used as the key.
Do you know how I should configure the "Field Values List"? BTW, is there any way to configure kloadgen not to randomly generate data in the event but always use the pre-defined data (including the key name in the map) in the "Field Values List"? Thanks.

**Comment by nuriaricofr** [comment]
Hi @billjliu ,
that function is not currently supported. Can you open an issue ?
